Examples/create-windows-online-cloud-image.ps1

```diff
@@ -48,7 +48,10 @@ $switchName = 'external'
 
 # The path were you want to create the config fille
 $configFilePath =".\config.ini"
-New-WindowsImageConfig -ConfigFilePath $configFilePath
```

Comment:
This was a decision before hand, to reset the image config file.
Please remove this code, as you change logic outside of the scope of the patch.